docs/examples/image_segmentation/mean_shift.jl

```diff
@@ -0,0 +1,89 @@
+# ---
+# cover: assets/meanshift.gif
+# title: Mean Shift Segmentation Algorithm
+# description: This demo shows how to use the meanshift algorithm to segment an image.
+# author: Ashwani Rathee
+# date: 2021-09-18
+# ---
+
+# Mean Shift segmentation is a local homogenization technique that is very useful
+# for damping shading or tonality differences in localized objects.
+# It replaces each pixel with the mean of the pixels in a range-r neighborhood 
+# and whose value is within a distance d.
+
+# K-means segmentation is another segmentation technique which is popular
+# but has two main issues:
+# -  We have specify a prior on the number of clusters(Number of K which is no of
+#    segments).
+# - It can be sensitive to value of k-means parameter.
```

Comment:
This seems not entirely surprising. Is it really worth making as a separate point?

It could also be stated that K-means tends to return clusters of approximately equal radius, which may not be how a human would do it.